generators/client-2/templates/src/main/webapp/app/admin/user-management/_user-management.state.ts

```diff
@@ -74,7 +74,6 @@ export function UserMgmntStateConfig($stateProvider) {
         }
     })
     .state('user-management.new', {
-        parent: 'user-management',
```

Comment:
declaring a parent using `name: 'parent.child'` *AND* having a `parent: ` property is ambiguous, and now throws an error